manifests/JetBrains/Toolbox/1.17.7018.0.yaml

```
@@ -0,0 +1,16 @@
+Id: JetBrains.Toolbox
+Publisher: Jet Brains
```

```suggestion
Publisher: JetBrains
```

They always use `JetBrains` (see https://www.jetbrains.com/company/).